Applications/Piano/AudioEngine.cpp

```diff
@@ -143,6 +147,37 @@ void AudioEngine::reset()
     m_previous_column = horizontal_notes - 1;
 }
 
+const String AudioEngine::set_recorded_sample(const StringView& path)
```

Comment:
`const` is kinda meaningless on a return type like this, and `String` is logically immutable anyway :)